src/helpers/http.ts

```diff
@@ -0,0 +1,7 @@
+export const safeResJson = (res: Response) => {
+  if (res.ok) {
+    return res.json()
+  } else {
```

Comment:
You don't need the `else` statement. Just throw the exception after the `if` block.

Reply:
you are right.
fixed now👌